src/fides/api/ctl/migrations/versions/392992c7733a_add_user_id_as_a_provided_identity_type.py

```diff
@@ -0,0 +1,39 @@
+"""Add user_id as a provided identity type
+
+Revision ID: 392992c7733a
+Revises: de456534dbda
+Create Date: 2023-01-20 15:33:24.084841
+
+"""
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "392992c7733a"
+down_revision = "de456534dbda"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    with op.get_context().autocommit_block():
+        print("hi")
```

Comment:
extra print statement

Reply:
🤦‍♀️ at least it wasn't more embarrassing

Reply:
😄 I know what you mean